src/ape/managers/config.py

```diff
@@ -216,7 +217,9 @@ def _plugin_configs(self) -> Dict[str, PluginConfig]:
             user_override = user_config.pop(plugin_name, {}) or {}
             if config_class != ConfigDict:
                 # NOTE: Will raise if improperly provided keys
-                config = config_class.from_overrides(user_override)  # type: ignore
+                config = config_class.from_overrides(  # type: ignore
```

Comment:
maybe a comment here like "Passing self in case loading plugin config depends on root config items" or something